closed/src/java.base/share/classes/openj9/internal/security/RestrictedSecurity.java

```diff
@@ -948,34 +774,52 @@ boolean isRestrictedServiceAllowed(Service service) {
                 return false;
             } else if (constraints.length == 0) {
                 // Allow this provider with no constraints.
+                if (debug != null) {
```

Comment:
Similar to https://github.com/ibmruntimes/openj9-openjdk-jdk17/pull/359#pullrequestreview-2131242787, this file should not have unnecessary differences relative to later versions. Please add the debug code before line 774, the blank line before line 870 and update the comment on line 892.

Reply:
Done.